oval_graph/command_line.py

```diff
@@ -23,4 +37,4 @@ def arf_to_html():
 
 
 if __name__ == '__main__':
-    arf_to_html()
+    globals()[sys.argv[1]](sys.argv[2:])
```

Comment:
I would remove this completely - `globals()[sys.argv[1]](sys.argv[2:])` is difficult to maintain and difficult to understand.